common/record.js

```diff
@@ -1,22 +1,77 @@
 (function () {
     'use strict';
 
-    angular.module('chaise.record.display', [])
+    angular.module('chaise.record.display', ['chaise.record.table'])
 
     .filter('trustedHTML', ['$sce', function($sce){
         return function(text) {
             return $sce.trustAsHtml(text);
         };
     }])
-
-    .directive('recordDisplay', [function() {
+     /* *Directive record-display is called to display related table as part of entity layout.
```

Comment:
Fix the indentation please. When you're changing code make sure the indentation is correct.

Also jsdoc only understands the following structure for comments:
```javascript
/**   DESCRIPTION HERE   */
```
Notice the two `*`s (without space in between). It's also cleaner if you leave the first line empty and also start each line with a `*`. If you're using atom, when you type `/**` the line before and then press enter it will generate a template of jsdoc which you can then edit.

```javascript
/**
 * DESCRIPTION
 * @param {string} param parameter's explanation
 */
function test(param) {}
```

`@param` syntax is not the way that you used it here. Take a look at [jsdoc documentation](http://usejsdoc.org/) for more information about the tags. I think we should add a very simple jsdoc explanation to our dev guides.